GVRf/Sample/gvr-video/src/org/gearvrf/video/VideoScript.java

```diff
@@ -632,6 +632,38 @@ public void run() {
             mainScene.getMainCameraRig().getOwnerObject()
                     .addChildObject(mPassThroughObject);
 
+            /*
+             * Selective objects to keep
+             */
+            mPlayPauseButton.getRenderData().getMaterial().setTexture("active_play", mActivePlay);
```

Comment:
I _know_ this sample is full of repeated calls like `mPlayPauseButton.getRenderData().getMaterial()`, but they really aren't free and I don't think we should be encouraging what amounts to a bad idea.

That is, I'd sure prefer it if you changed code like

            mPlayPauseButton = new GVRSceneObject(gvrContext,
                    gvrContext.createQuad(0.7f, 0.7f), mInactivePause);
            mPlayPauseButton.getTransform().setPosition(0.0f, -0.8f, -8.0f);
            mPlayPauseButton.getRenderData().setRenderingOrder(
                    GVRRenderingOrder.TRANSPARENT + 1);
            mPlayPauseButton.getRenderData().setOffset(true);
            mPlayPauseButton.getRenderData().setOffsetFactor(-1.0f);
            mPlayPauseButton.getRenderData().setOffsetUnits(-1.0f);

to 

            mPlayPauseButton = new GVRSceneObject(gvrContext,
                    gvrContext.createQuad(0.7f, 0.7f), mInactivePause);
            mPlayPauseButton.getTransform().setPosition(0.0f, -0.8f, -8.0f);
            GVRRenderData playPauseRenderData = mPlayPauseButton.getRenderData();
            playPauseRenderData.setRenderingOrder(
                    GVRRenderingOrder.TRANSPARENT + 1);
            playPauseRenderData.setOffset(true);
            playPauseRenderData.setOffsetFactor(-1.0f);
            playPauseRenderData.setOffsetUnits(-1.0f);

throughout `onInit()`. Otoh, that does go far beyond fixing a bug, so I won't insist on it.